about.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="stylesheet" href="about.css">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="first-header">
+    <a href=""><img src="imgs/facebook.png" alt=""></a>
+    <a href=""><img src="imgs/twitter.png" alt=""></a>
```

Comment:
- [ ] all of your pages should be accessible, currently, you have some empty  `alt` attributes kindly add meaningful alts for each `img` tag you have